02_bubble/eastrd/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+func bubble(s []int) []int {
+	/*
+		Returns a sorted copy of slice s using bubble sort
```

Comment:
Typically `go` function comments are written write above the function starting with the function name.
In you case that would be
```go
// bubble returns a sorted copy of slice s using bubble sort
func bubble(s []int) []int {
```